resources/views/accessories/checkout.blade.php

```diff
@@ -53,7 +53,7 @@
 
           @include ('partials.forms.edit.user-select', ['translated_name' => trans('general.select_user'), 'fieldname' => 'assigned_to'])
 
-
+{{--this if statement doesn't bring up the requireAcceptance() label or trigger the proper email. Haven't figured out why yet.--}}
              @if ($accessory->requireAcceptance() || $accessory->getEula() || ($snipeSettings->slack_endpoint!=''))
```

Comment:
Are you sure the category the accessory you're testing this with has a the acceptance stuff selected?

Reply:
This seems to be working fine for me when I test.

<img width="1004" alt="Screen Shot 2022-02-11 at 8 21 04 PM" src="https://user-images.githubusercontent.com/197404/153696304-0349e1a3-71e1-44cb-96a2-24d352ed6e78.png">

<img width="862" alt="Screen Shot 2022-02-11 at 8 21 56 PM" src="https://user-images.githubusercontent.com/197404/153696313-520abb43-7a58-4051-86ed-f7d07f43e31d.png">

Reply:
I did not have them required under categories. Thanks